Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -2185,10 +2183,10 @@ - (BOOL)accessibilityIsAttributeSettable:(NSString*)attributeName
 
     return NO;
 }
+ALLOW_DEPRECATED_IMPLEMENTATIONS_END
```

Comment:
I found some `.mm` files doing this (i.e. not putting the `*END` macro right after definition opening brace `{`, but all the way after the definition closing brace `}`, incl. when the function body is long. It avoids indentation warning (which might be false positive) around the opening brace `{`.